src/auth/UserRequest.cc

```diff
@@ -132,6 +132,18 @@ Auth::UserRequest::denyMessage(char const * const default_message) const
     return getDenyMessage();
 }
 
+bool Auth::UserRequest::authenticated() const
+{
+    if (user() != nullptr && user()->credentials() == Auth::Ok)
+    {
+        debugs(29, DBG_DATA, "user authenticated.");
```

Comment:
DBG_DATA is for data dumps. Please use level 7, do not repeat what debugs() says already, and remove decorations:

```suggestion
        debugs(29, 7, "yes");
```

Same for the other one.

Reply:
done

Reply:
It was not quite "done", and I adjusted this code further in branch commit [04eeac3](https://github.com/squid-cache/squid/pull/1679/commits/04eeac3c13882664268658a9e9e52adb27582265).